### PR TITLE
Test library without dummy rails app

### PR DESCRIPTION
**Pull request by bbuchalter**

### CODE REVIEW

File: ruby/bitemporality.gemspec [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  spec.bindir        = "exe"
+  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.require_paths = ["lib"
+  ]
+  spec.add_dependency "activerecord", "~> 5.0.0"
+
+  spec.add_development_dependency "bundler", "~> 1.17"
+  spec.add_development_dependency "rake", "~> 10.0"
+  spec.add_development_dependency "rspec", "~> 3.0"
+  spec.add_development_dependency "sqlite3", '~> 1.3.13'
```
Comment (wenley) [inline ruby/bitemporality.gemspec]: Confirming: Are these the lines that generated the `Gemfile.lock` changes?

File: ruby/bitemporality.gemspec [hunk 2]
```diff
@@ -0,0 +1,43 @@
+lib = File.expand_path("../lib", __FILE__)
```
Comment (wenley) [inline ruby/bitemporality.gemspec]: Is this file handwritten? I'm guessing not based on the funkiness of its contents -> what command did you run to generate it?

It seems like `bundle gem <gem-name>` can auto-generate one, but the PR doesn't include other things like a license file.

Comment (bbuchalter) [reply ruby/bitemporality.gemspec]: I wanted the change to be as small as possible, so I just cherry-picked in this change from a `bundle gem` command I ran elsewhere.

File: ruby/bitemporality.gemspec (hunk 1 above)
Comment (bbuchalter) [reply ruby/bitemporality.gemspec]: Yes.

**Pull request merged**